bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -589,6 +620,24 @@ public void onAnimationEnd(Animator animation) {
     }
 
     /**
+     * Controls the behavior on back button press when the state is {@link State#EXPANDED}.
+     *
+     * @param state {@link State#PEEKED} to show the peeked state on back press or {@link
+     *              State#HIDDEN} to completely hide the Bottom Sheet. Default is {@link  State#HIDDEN}.
+     */
+    public void setDismissMode(State state) {
```

Comment:
It makes more sense to take this parameter in as a boolean as not all values for `State` are allowed. e.g. `State.EXPANDED`.